execution-service/src/main/java/org/ihtsdo/rvf/execution/service/ValidationVersionLoader.java

```diff
@@ -119,8 +119,7 @@ public void loadProspectiveVersion(ValidationStatusReport statusReport, MysqlExe
 		}
 
 		if (!validationConfig.isRf2DeltaOnly() && !checkDeltaFilesExist(validationConfig.getLocalProspectiveFile())) {
-			final String schemaName = prospectiveVersion.startsWith(RVF_DB_PREFIX) ? prospectiveVersion : RVF_DB_PREFIX + prospectiveVersion;
-			releaseDataManager.insertIntoProspectiveDeltaTables(schemaName, executionConfig);
+			validationConfig.setMissingRf2DeltaFromRelease(true);
```

Comment:
@QuyenLy87  I think there are other assertions not for "release-type-validation" also require delta. e.g component-centric validations for current authoring cycle only. If the delta is empty in the mysql db then these assertions will always pass which is bad.